templates/server/src/main/resources/archetype-resources/core/src/test/java/__packageInPathFormat__/general/common/base/SpringBootAppTest.java

```diff
@@ -0,0 +1,18 @@
+package ${package}.general.common.base;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+
+import com.devonfw.module.test.common.base.ModuleTest;
+
+/**
+ * This test verifies that {@link SpringBootApp} is able to startup.
+ */
+@SpringBootTest
+public class SpringBootAppTest extends ModuleTest {
```

Comment:
Looking here a second time:
* Dont we have to specify the SpringBootApp?
* Actually all we need to do is extend `ApplicationComponentTest` and we do not need any annotation anymore.